build/broccoli/build-packages.js

```diff
@@ -74,7 +74,8 @@ function topsortPackages() {
 function treeForPackage(packagePath) {
   let packageTree = buildPackage({
     projectPath: packagePath,
-    tsconfigPath: TSCONFIG_PATH
+    tsconfigPath: TSCONFIG_PATH,
+    include: ['lib/**/*.ts', 'index.ts']
```

Comment:
@stefanpenner OK, sorry for the stream of consciousness. This path is fed to Broccoli. Previously in `@glimmer/build` it was sucking in the entire directory in the TypeScript plugin, including `tmp` and `dist`. You can guess what happened next. So now it filters only for `.ts` files in `src`. But in our case, the TS files live in the root dir, not `src`—and they don't have `dist` or `tmp` directories.